config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
   resources :talks
+  post "/reset_password", controller: :reset_password, action: :create
```

Comment:
It would be a little more canonical to do `resources :password_resets, only: :create` and rename the files involved.